src/act.c

```diff
@@ -2182,12 +2182,12 @@ void save_file_w( )
 	sprintf(path_work, "/ram/work.sav");
 	sprintf(path_item, "/ram/item_wk.sav");
 	
-#elif defined(_TINSPIRE)
-	sprintf(path_config, "./save/config.tns");

```

Comment:
weird, don't know how those lines disappeared, they should be there... I'll amend it.